.github/workflows/_runner-determinator.yml

```diff
@@ -34,11 +34,120 @@ jobs:
       ISSUE_NUMBER: ${{ inputs.issue_number }}
       USERNAME: ${{ inputs.user_name }}
     steps:
-      - name: Checkout PyTorch
-        uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
-        with:
-          fetch-depth: 1
-          submodules: true
+      # - name: Checkout PyTorch
+      #   uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
+      #   with:
+      #     fetch-depth: 1
+      #     submodules: true
+
+      - name: Hardcode runner-determinator script
```

Comment:
How about adding a comment here explaining that this is temporary and that the embedded file should match the python script it was sourced from?

Reply:
Good point, I added some comments and a TODO comment too.